A66-otel-stats.md

```diff
@@ -0,0 +1,542 @@
+    *Attributes*: grpc.method, grpc.target, grpc.status <br>
+    *Type*: Histogram (Latency Buckets) <br>
+
+#### Server Instruments
+
+*   **grpc.server.call.started** <br>
+    The total number of RPCs started, including those that have not completed. <br>
+    *Attributes*: grpc.method <br>
+    *Type*: counter <br>
+    *Unit*: {call} <br>
+*   **grpc.server.call.sent_total_compressed_message_size** <br>
+    Total bytes (compressed but not encrypted) sent across all response messages (metadata excluded) per RPC; does not include grpc or transport framing bytes. <br>
+    *Attributes*: grpc.method, grpc.status <br>
+    *Type*: Histogram (Size Buckets) <br>
+*   **grpc.server.call.rcvd_total_compressed_message_size** <br>
+    Total bytes (compressed but not encrypted) received across all request messages (metadata excluded) per RPC; does not include grpc or transport framing bytes. <br>
+    *Attributes*: grpc.method, grpc.status <br>
+    *Type*: Histogram (Size Buckets) <br>
+*   **grpc.server.call.duration** <br>
+    This metric aims to measure the end2end time an RPC takes from the server transport’s (HTTP2/ inproc / cronet) perspective. <br>
+    Start timestamp - After the transport knows that it's got a new stream. For HTTP2, this would be after the first header frame for the stream has been received and decoded. Whether the timestamp is recorded before or after HPACK is left to the implementation. <br>
+    End timestamp - Ends at the first point where the transport considers the stream done. For HTTP2, this would be when scheduling a trailing header with END_STREAM to be written, or RST_STREAM, or a connection abort. Note that this wouldn’t necessarily mean that the bytes have also been immediately scheduled to be written by TCP. <br>
+    *Attributes*: grpc.method, grpc.status <br>
+    *Type*: Histogram (Latency Buckets) <br>
+
+### OpenTelemetry Plugin Architecture
+
+This section describes a `CallTracer` approach to collect the client and server
+per-attempt/call metrics. Implementations are free to choose different ways of
+representing/naming the classes and methods described here as long as the
+overall capabilities remain equivalent.
+
+A CallTracer is a class that is instantiated for every call. This class has
+various methods that are invoked during the lifetime of the call. On the
+client-side, the CallTracer knows about multiple attempts on the same call (due
+to retries or hedging), and creates a `CallAttemptTracer` object for each
+attempt, and the `CallAttemptTracer` gets invoked during the lifetime of the
+attempt. On the server-side, we have an equivalent `ServerCallTracer`. (There is
+no concept of an attempt on the server-side.)
+
+The OTel plugin will basically be a way of configuring CallTracer factories on
+gRPC channels and servers.
```

Comment:
Try to avoid words like "basically".  Cleaner is: `The OTel plugin will configure CallTracer factories on gRPC channels and servers.`

Also, should we avoid `OTel` as a lingo-ish term and use `Open Telemetry` everywhere instead?

Reply:
https://opentelemetry.io/docs/specs/otel/#project-naming It's official :D

Reply:
> Try to avoid words like "basically". Cleaner is: The OTel plugin will configure CallTracer factories on gRPC channels and servers.

done

Reply:
Nit: I agree with @dfawley. Although, `OTel` is an official acronym imo using project name `OpenTelemetry` instead of the acronym in gRFC sounds better.

Reply:
done